doc/include/api_stubs.inc

```diff
@@ -129,45 +103,32 @@ current [file .critcl] file.
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::api] [method extheader] [opt [arg file]...]]
 
-This method is similar [cmd {::critcl::api header}], in that it
-notifies critcl of companion header files which have to be exported
-together with the generated stubs headers.
-
-[para] The difference is that these headers will be expected to exist
-in the external development environment.
-
-As such they will be [term {#include}]d in the generated header for
-the package, but not copied to the package header directory. Nor are
-they allowed to be glob patterns, as critcl has no context, i.e
-directory, in which to expand such patterns.
+Like [cmd {::critcl::api header}], but each [arg file] should exist in the
+external development environment.  An [term {#include}] directive is added to
+[file [var foo]Decls.h], but [arg fiile] is not copied to the package header
```

Comment:
`fiile` -> `file`

Reply:
Done with commit 29380521d0873ea1dcc5f8cdff6742b20764473f (Branch `documentation` in my repo, not seen by the PR).